src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -2895,4 +2895,88 @@ public void startFakeRepairSchedule() {
           Response.Status.NOT_FOUND);
     });
   }
+
+  @And("^I add 11 and abort the most recent 10 repairs for cluster \"([^\"]*)\" and keyspace \"([^\"]*)\"$")
+  public void addAndAbortRepairs(String clusterName, String keyspace) throws Throwable {
+    synchronized (BasicSteps.class) {
+      testContext.TEST_CLUSTER = (String) clusterName;
+      Set<String> tables = Sets.newHashSet();
+      testContext.addClusterInfo(clusterName, keyspace, tables);
+      HashMap<String, String> params = Maps.newHashMap();
+      params.put("clusterName", clusterName);
+      params.put("keyspace", keyspace);
+      params.put("owner", "test_user");
+
+      RUNNERS.parallelStream().forEach(runner -> {
+        Integer iter = 1;
+        while (iter <= 11) {
+          Response response = runner.callReaper("POST", "/repair_run", Optional.of(params));
+          String responseData = response.readEntity(String.class);
+          Assertions
+              .assertThat(response.getStatus())
+              .isEqualTo(Response.Status.CREATED.getStatusCode())
+              .withFailMessage(responseData);
+          String id = "";
+          try {
+            id = new ObjectMapper().readValue(responseData, String.class);
+            testContext.addCurrentRepairId(UUID.fromString(id));
+          } catch (Throwable e) {
+            Assertions.fail("response deserialisation failed");
+          }
```

Comment:
You should rather catch a RuntimeException and display its content in the output, because right now we cannot know what's the underlying error

```suggestion
          } catch (RuntimeException e) {
            LOG.error("response deserialisation failed", e);
            LOG.error("Response data was: {}", responseData);
          }
```

Reply:
Doing:

```
            LOG.error("response deserialisation failed", e);
            LOG.error("Response data was: {}", responseData);
            Assertions.fail("response deserialisation failed");
```

Because we still want it to fail right?